packages/@uppy/audio/src/RecordingScreen.tsx

```diff
@@ -126,3 +126,7 @@ export default function RecordingScreen(props: RecordingScreenProps) {
     </div>
   )
 }
+
+RecordingScreen.defaultProps = {
```

Comment:
We shouldn't use default props anymore. You can just do it in JS. Also lint CI is failing.

Reply:
Do you want to open a PR to change the linter settings regarding that? Currently the linter requires defining `defaultProps`

Reply:
I think we can simply remove the rule, there's no replacement.

Reply:
We should probably have a rule to forbid use of `defaultProps`, so the codebase is consistant